src/egpg.sh

```diff
@@ -192,6 +192,7 @@ config() {
 
     export GNUPGHOME
     export GPG_TTY=$(tty)
+    export EGPG_DEBUG=$DEBUG
```

Comment:
Why introduce a new variable?

Reply:
I want to test for DEBUG in all python scripts. $DEBUG is not already exported. So did it this way as variable name DEBUG itself as it is very common variable used by other programs also I didn't wanted it to interfere with the already running programs.

Reply:
I had kind of misinterpreted working of calls in bash, I knew that environments are only propagated downwards but I forgot to consider that each call in Bash launches a new subprocess and hence exporting a variable won't affect the main shell. Have corrected this and now  we are using variable $DEBUG only.